indra/newview/llpanelemojicomplete.cpp

```diff
@@ -50,6 +51,7 @@ LLPanelEmojiComplete::Params::Params()
 LLPanelEmojiComplete::LLPanelEmojiComplete(const LLPanelEmojiComplete::Params& p)
 	: LLUICtrl(p)
 	, mAutoSize(p.autosize)
+	, mNoScroll(p.noscroll)
```

Comment:
if noscroll is optional, it's better to have a default value here, then do an 'if (p.noscroll.isProvided()) mNoScroll = p.noscroll;'
Or noscroll itself can have a default value

Reply:
It works right similar to autosize
The default value is default for the value type (false for bool)
We should specify the default value for the parameter when it differs from the default value of the type
No needs to specify it explicitly here